utils/fileIOUtils_test.go

```diff
@@ -0,0 +1,148 @@
+// +integration
+package utils
+
+import (
+	"testing"
+	"os"
+	"io/ioutil"
+)
+
+const testKeysFilePath string = ApplicationRoot + "/test_keys_config.yaml"
```

Comment:
Hardcoding '/' character in the file path won't work on Windows because Windows uses back-slashes. There should be an equivalent of File.Separator (Java) on Go as well.

Reply:
Fixed.